examples/material_gallery/lib/demo/dialog_demo.dart

```diff
@@ -0,0 +1,37 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/widgets.dart';
+
+const String _dialogText =
+  "Use dialogs sparingly because they are interruptive. Their sudden appearance "
+  "forces users to stop their current task and focus on the dialog content. "
+  "Alternatives to dialogs include menus or inline expansion, both of which "
+  "maintain the current context.";
+
+class DialogDemo extends StatelessComponent {
+  DialogDemo({ Key key }) : super(key: key);
+
+  Widget build(BuildContext context) {
+    final ThemeData theme = Theme.of(context);
+    return new Dialog(
+      title: new Text("This is a Dialog"),
+      content: new Text(
+        _dialogText,
+        style: theme.text.subhead.copyWith(color: theme.text.caption.color)
```

Comment:
I 'd actually changed Dialog to do it and then checked with the Material Design folks. Their feedback:

> ..this particular example is trying to decrease the prominence of it because they want users to focus on the main question (which is in the dialog title). Since the body text there is mostly a deeper explanation of the title question, its been given the secondary color (54% black).
WRT the color, I think that particular example is trying to decrease the prominence of it because they want users to focus on the main question (which is in the dialog title). Since the body text there is mostly a deeper explanation of the title question, its been given the secondary color (54% black).

I think we do need an easier way to say: give me lower contrast text color per the current theme.